chat-plugins/othermetas.js

```diff
@@ -311,24 +311,22 @@ exports.commands = {
 	},
 	tiershifthelp: [`/ts OR /tiershift <pokemon> - Shows the base stats that a Pokemon would have in Tier Shift.`],
 
-	'!scalemons': true,
-	scale: 'scalemons',
-	scalemons: function (target, room, user) {
```

Comment:
Uh no???????????

Reply:
Add a new command, don’t replace the old one. That’s the whole reason why we havd this chat plugin.